cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -1620,6 +1623,26 @@
             (_ Unit (emit (MInst.VecTbl dst rn rm is_extension))))
         dst))
 
+;; Helper for emitting `MInst.VecTbl2` instructions.
+;; - 2 register table vector lookups require consecutive table registers;
+;;   we satisfy this constraint by hardcoding the usage of v30 and v31.
+;; - Make sure that both args are in virtual regs, since it is not guaranteed
+;;   that we can get them safely to the temporaries if either is in a real
+;;   register.
+(decl vec_tbl2 (Reg Reg Reg bool Type) Reg)
+(rule (vec_tbl2 rn rn2 rm is_extension ty)
+      (let (
+            (temp WritableReg (writable_vreg 30))
+            (temp2 WritableReg (writable_vreg 31))
+            (dst WritableReg (temp_writable_reg $I8X16))
+            (rn_1 Reg (ensure_in_vreg rn ty))
```

Comment:
ISLE supports shadowing variable bindings now (as of end of last week, thanks to Trevor!) so this can be just `rn` (and `rn2` below) I think.